src/orca/services/synapse/ops.py

```diff
@@ -57,3 +61,73 @@ def monitor_evaluation_queue(self, evaluation_id: str) -> bool:
         )
         submissions_num = sum(1 for submission in received_submissions)
         return submissions_num > 0
+
+    def trigger_indexing(self, syn: Synapse, synapse_view: str) -> None:
+        """
+        Trigger indexing in Synapse.
+
+        Arguments:
+            syn: A Synapse object.
+            synapse_view: The Synapse ID of the view.
+        """
+        syn.tableQuery(f"select * from {synapse_view} limit 1")
+
+    def get_submissions(
+        self, submission_view: str, submission_status: str = "RECEIVED"
+    ) -> List[str]:
+        """
+        Get all submissions with desired submission status in a Synapse
+        submission view.
+
+        Arguments:
+            submission_view: The Synapse view to get submissions from.
+            submission_status: The submission status to filter for.
+                               Defaults to "RECEIVED".
+
+        Returns:
+            A list of submission IDs.
+        """
+
+        # Gain access to Synapse API
+        syn = self.client
+
+        # Trigger indexing
+        self.trigger_indexing(syn, submission_view)
+
+        # Get all submissions for the given ``submission_view``
+        query_results = syn.tableQuery(
+            f"select * from {submission_view} where status = '{submission_status}'"
+        )
+
+        submission_ids = query_results.asDataFrame()["id"].tolist()
+
+        return submission_ids
+
+    def update_submissions_status(
```

Comment:
For simplicity should this only take lists of IDs? We can always provide a list of one if need be

Reply:
Yes I think it would be good to limit the scope of this function and run the loop outside of it; have it run in airflow. If this works for you I'll go ahead and update this along with its tests.

Reply:
Sounds good!